src/panels/lovelace/cards/hui-conditional-card.ts

```diff
@@ -49,15 +44,43 @@ class HuiConditionalCard extends HTMLElement implements LovelaceCard {
         return hass.states[c.entity].state !== c.state_not;
       });
 
-    if (visible && this.card) {
-      this.card.hass = hass;
+    if (this._card) {
+      this._card.hass = this._hass;
     }
 
     this.style.setProperty("display", visible ? "" : "none");
   }
 
-  public getCardSize() {
-    return computeCardSize(this.card);
+  public getCardSize(): number {
+    return computeCardSize(this._card);
+  }
+
+  public setConfig(config: Config): void {
+    if (
+      !config.card ||
+      !config.conditions ||
+      !Array.isArray(config.conditions) ||
+      !config.conditions.every(
+        (c) =>
```

Comment:
This is just a personal preference, not sure if best practice but using the singular version of what the array is so in this card `condition`

Reply:
agreed